### DISCUSSION TITLE
please return releases of strawberry.appimage :'(

### DISCUSSION THREAD

**Discussion by NektoNektovich** [strawberrymusicplayer/strawberry#1150] [Q&A]
it's very sad that there is no strawberry.appimage, since my current distribution does not install any of the rpm packages, and the repository has a very old version of strawberry 0.93. I still have strawberry-0.93.appimage and it works, but auto-import of tags for filling does not work, and this is the function I need :(

**Comment by jonaski** [comment]
What distribution?

**Reply by NektoNektovich** [reply]
> What distribution?

Exotic - 
Simply linux p10.1

**Comment by NektoNektovich** [comment]
moreover, I'm not asking you to collect strawberry.appimage every new version, but even 1 out of 3 or at least 1 out of 5 would be happy 🤗

**Reply by jonaski** [reply]
See https://forum.strawberrymusicplayer.org/topic/1094/appimages-for-linux
